src/main/java/cms/gov/madie/measure/services/MeasureSetService.java

```diff
@@ -323,4 +323,37 @@ public List<Measure> getRecentMeasuresByMeasureSetId(List<String> measureSetIds)
     }
     return mostRecentMeasures;
   }
+
+  /**
+   * Change ownership in MeasureSet
+   *
+   * @param measureSetId - the MeasureSet that needs change of ownership
+   * @param userId - new owner
+   * @param retainShareAccess - keep SHARED_WITH if true, otherwise remove
+   * @param conductedBy - the user that performs the ownership change
+   * @return
+   */
+  public MeasureSet changeOwnership(
+      String measureSetId, String userId, boolean retainShareAccess, String conductedBy) {
+    Optional<MeasureSet> OptionalMeasureSet = measureSetRepository.findByMeasureSetId(measureSetId);
```

Comment:
naming convention
```suggestion
    Optional<MeasureSet> optionalMeasureSet = measureSetRepository.findByMeasureSetId(measureSetId);
```

Reply:
@adongare Ok typo, thanks, will change it with other refactoring stuff. Thanks!